compose/__init__.py

```diff
@@ -1,4 +1,8 @@
 from __future__ import unicode_literals
 from .service import Service  # noqa:flake8
 
+from collections import namedtuple
+
 __version__ = '1.3.0dev'
+
+VolumeFrom = namedtuple("VolumeFrom", 'source mount_option')
```

Comment:
Yeah that's right.. I moved it back to project.py — i'll see if janky is happy.. I'm on a EDGE network right now, it's taking for ever to build/test it on my laptop :cry: .